### PR TITLE
Delete MyImages, Add logos for jobs container

### PR DESCRIPTION
**Pull request by RCAVelez**
Deleted MyImages folder
Added images to images folder
Presenting images in jobs container
Created way to animate scroll to section when link clicked

### CODE REVIEW

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html]: Please integrate and automate some formatter like Prettier, I don't want to continue to nitpick on formatting.

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js]: just curious, do you need to animate both html and body? I feel one of them should be enough.

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js]: I see this function to animate reused 4 times in this file. I think following the DRY principle and creating a utility function to animate might clean things up a bit.

```
const animateScrollTop = (targetClass) => {
  $("html").animate({ scrollTop: $("." + targetClass).offset().top }, 2000);
};

// From each click listener, call animateScrollTop().
```

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css]: again, I recommend hooking up an auto formatter

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css]: wrapping all the CSS rules in a media query seems odd to me. This means that accessing from a phone will load no css rules.
🠪 1. is the goal to only load CSS rules for tablet/desktop because mobile styling is handled in a separate PR?
🠪 2. OR, is mobile styling not covered at all in your starter projects?

**Review by genedwards — CHANGES_REQUESTED** [review]
Please add detailed PR description so that I know the intent of this PR.

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html]: Branch RCAVelez-redesignIndex has addressed these issues, once that branch is approved I will rebase and merge. But this problem will propagate through each branch for now because of the timing of PRs

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css]: Branch RCAVelez-redesignIndex has removed media query.

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css]: yup, addressed in branch RCAVelez-redesignIndex, will propagate through further PRs

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js]: done

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js
Comment (RCAVelez) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js]: done

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js:17]: do you want to keep this console log?

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html
Comment (genedwards) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:15]: onclick attributes are generally considered bad practice because of mixing of template and logic, more difficult to test (there's probably a bunch more, just Google it)...but if you won't be building on this code, I'm ok with it
I'll leave it up to you

**Review by genedwards — APPROVED** [review]
other than the small comments, LGTM

**Pull request merged**
